src/ConnectionStates/PlayingState.cs

```diff
@@ -28,7 +28,7 @@ public PlayingState(Session session)
             if (session.Thing != null)
             {
                 var behavior = session.Thing.Behaviors.FindFirst<PlayerBehavior>();
-                this.playerBehavior = new WeakReference<PlayerBehavior>(behavior);
+                this.playerBehavior = new Utilities.WeakReference<PlayerBehavior>(behavior);
```

Comment:
nit: Remove our custom one instead?  Created #40 to track or discuss as necessary.